webapp/app.py

```diff
@@ -1062,3 +1063,15 @@ def cache_headers(response):
         response.cache_control.no_store = True
 
     return response
+
+
+def date_has_passed(date_str):
+    try:
+        date = datetime.strptime(date_str, "%Y-%m-%d")
+        present = datetime.now()
+        return present > date
+    except ValueError:
+        return False
+
+
+app.add_template_filter(date_has_passed)
```

Comment:
We tend to use [template context](https://github.com/canonical/ubuntu.com/blob/main/webapp/app.py#L271) but I don't see anything wrong doing it this way